ui.apps/src/main/content/jcr_root/apps/saas-aem-module/clientlibs/wknd-sample/css/styles.less

```diff
@@ -35,14 +35,29 @@
   }
 }
 @media only screen and (min-width: 769px) {
-  .search .saas-container_results {
+  .search .saas-container__results {
     display: grid !important;
```

Comment:
Can we avoid using important here 🙈

Reply:
@Ivana22 I know, that's ugly hey ;-)

This was set because there is a set display block programmatically in the code. I will see if I can change to using a class --is-visible instead.